### DISCUSSION TITLE
Can I use react-native-web in WordPress?

### DISCUSSION THREAD

**Discussion by Stevemoretz** [necolas/react-native-web#1992] [Q&A] [answered]
Hi I use react native for mobile application development.
I use react for web development, using Web Components.

So I can use react inside WordPress, Laravel ,...

I have never used react-native-web yet.

Can it be used as web component?
I want to inject some components here and there I don't want it to be the whole website.

Is it possible?
If yes how please?

**Comment by necolas** [comment, marked as answer]
You can use `react-native-web` anywhere you can install npm packages and use `react-dom`. You should refer to Wordpress documentation for using JavaScript packages like `react-dom`.
